build/building/packaging.md

```diff
@@ -113,122 +127,149 @@ your Dockerfile, and should be the first line in Dockerfiles.
 > release of the version 1 syntax. BuildKit automatically checks for updates of
 > the syntax before building, making sure you are using the most current version.
```

Comment:
I know not in the scope of your changes, but how about making this a 'tip' callout? Lots of blue callouts on le page

Reply:
Good idea!